### DISCUSSION TITLE
Minimum noise multipler for img2img to 0

### DISCUSSION THREAD

**Discussion by HarrisTerry** [vladmandic/sdnext#802] [Ideas]
noise multiplier for im2img gets more photorealism if it can go all the way to 0 instead of 0.5, right now I do some f12 trick and change the script temporarily to min 0, using 0.2 makes the photos look more real but the current minimum is 0.5, this would be a wonderful tweak. 

Thank you very much.

**Comment by therug** [comment]
I'd suggest changing the default values in ui-config.json, e.g.
```
    "img2img/CFG Scale/value": 6.0,
    "img2img/CFG Scale/minimum": 1.0,
    "img2img/CFG Scale/maximum": 30.0,
    "img2img/CFG Scale/step": 0.5,
```

**Reply by vladmandic** [reply]
aren't those already default values?
in `modules/ui.py`: `cfg_scale = gr.Slider(minimum=1.0, maximum=30.0, step=0.5, label='CFG Scale', value=6.0, elem_id="img2img_cfg_scale")`

ui-config.json is auto-created from actual ui components.

**Reply by therug** [reply]
Sorry, yes, my reply was directed at @SadisticLeprechaun—who could alter those values locally to match the values wished for, I thought.

**Comment by HarrisTerry** [comment]
From what someone showed me, the minimum used to be 0.5, but to adjust it
below that I had to use F12, click the slider, change the min value in a
script, and then could use it at 0.2.  Works great though and many thanks!
Didn't know there were other ways to achieve the setting.

**Comment by OrganicBeej** [comment]
> I'd suggest changing the default values in ui-config.json, e.g.
> 
> ```
>     "img2img/CFG Scale/value": 6.0,
>     "img2img/CFG Scale/minimum": 1.0,
>     "img2img/CFG Scale/maximum": 30.0,
>     "img2img/CFG Scale/step": 0.5,
> ```

Hi I also wanted to edit this inside the ui-config.json file, I have done so but it doesn't seem to change it from 0.5 in 'Noise multiplier for img2img'?

Any advice appreciated :)

**Comment by HarrisTerry** [comment]
Mine goes as low as I want now :)  I think this was fixed!
